pkg/reporter/interface.go

```diff
@@ -49,7 +49,7 @@ type ReportOptions struct {
 	IncludePassedRules bool
 
 	// MinSeverity filters results by minimum severity level
-	MinSeverity models.Severity
```

Comment:
if possible, we can have it a proper struct. It'll keep it consistent everywhere